tests/test_click_events.py

```diff
@@ -0,0 +1,31 @@
+def setup_app(app):
+    from lona.html import Button
+    from lona import LonaView
+
+    @app.route('/')
+    class MyLonaView(LonaView):
+        def handle_request(self, request):
+            self.show(Button('click me!'))
+            self.await_click()
+            self.show('SUCCESS')
+
+    app.settings.SESSIONS = False
+
+
+async def test_rendering(lona_app_context):
+    from playwright.async_api import async_playwright
+
+    context = await lona_app_context(setup_app)
+
+    async with async_playwright() as p:
+        browser = await p.chromium.launch()
+        page = await browser.new_page()
+        await page.goto(context.get_url())
+
+        assert await page.inner_text('#lona') != 'SUCCESS'
+
+        await page.click('button')
+
+        assert await page.inner_text('#lona') == 'SUCCESS'
```

Comment:
It's better to use methods that can wait for desired state.
```suggestion
        await page.wait_for_selector('#lona:has-text("SUCCESS")')
```

For example if I add `self.sleep(1)` before `self.show('SUCCESS')` in view handler, than suggested variant will pass, but initial one will fail.

Reply:
Ouh! :) Good to know! To be honest: This test was a quick-and-dirty proof-of-concept, and i was very happy that it does something.

Reply:
fixed

Reply:
I know that it's a POC. But I had a lot of troubles with that, so I had to say that. Sorry :)